tests/unit/misc/test_guiprocess.py

```diff
@@ -77,7 +77,24 @@ def test_start_verbose(proc, qtbot, message_mock, py_proc):
     assert msgs[1].level == usertypes.MessageLevel.info
     assert msgs[0].text.startswith("Executing:")
     assert msgs[1].text == "Testprocess exited successfully."
-    assert bytes(proc._proc.readAll()).rstrip() == b'test'
+    assert qutescheme.spawn_output == proc.spawn_format(0, 0, stdout="test")
+
+
+def test_start_output(proc, qtbot, message_mock, py_proc):
```

Comment:
This looks like it's the same as `test_start_verbose`?

Reply:
oh whoops, I was going to figure out how to test for opening the window, however as there are tests specific to opening the window, and the content of the window is essentially tested for in the other tests, it seemed superfluous. I accidentally left a relic of thought in, it's now removed!